.github/workflows/test.yaml

```diff
@@ -111,3 +111,14 @@ jobs:
       date: ${{ inputs.date }}
       sha: ${{ inputs.sha }}
       script: ci/test_wheel_dask.sh
+  wheel-tests-integrations:
+    secrets: inherit
+    uses: rapidsai/shared-workflows/.github/workflows/wheels-test.yaml@main
+    with:
+      build_type: ${{ inputs.build_type }}
+      branch: ${{ inputs.branch }}
+      date: ${{ inputs.date }}
+      sha: ${{ inputs.sha }}
+      script: ci/test_wheel_integrations.sh
+      # Test latest CUDA version with all Python versions
+      matrix_filter: group_by(.CUDA_VER|split(".")|map(tonumber)|.[0]) | map(max_by(.CUDA_VER|split(".")|map(tonumber))) | group_by(.PY_VER) | map(.[0])
```

Comment:
What sort of testing matrix do you want here?  
I would interpret the comment as testing on 4 python versions, all with CUDA 13.0.1, on arm and amd64, but this filter is not doing that, and also isn't guaranteed to grab the latest CUDA anything because it's position dependent.